py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -315,6 +315,17 @@ def aten_ops_matmul(
         args[1],
     )
 
+@dynamo_tensorrt_converter(torch.ops.aten.bmm.default)
```

Comment:
Put decorator in above function + rebase

Reply:
thanks!